src/main/java/com/revature/domain/Eval.java

```diff
@@ -35,92 +35,162 @@
 	@JoinColumn(name="e_batch")
 	private Batch batch;
 
-	public Eval() {}
+	@OneToMany(mappedBy="eval")
+	private List<QuestionEval> questions;
+	
+	@OneToMany(mappedBy="eval")
+	private List<EvalComment> comments;
 	
-	public Eval(Integer id, Integer week, Date date, EvalType evalType, Person trainee, Batch batch) {
+	public Eval() {}
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Add a nested comment explaining why this method is empty, throw an UnsupportedOperationException or complete the implementation. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.revaturelabs.com/coding_rules#rule_key=squid%3AS1186)